include/bl_crypto.rst

```diff
@@ -0,0 +1,51 @@
+.. _doc_bl_crypto:
+
+Bootloader crypto
+#################
+
+The bootloader crypto library is the cryptography library that is used by the :ref:`bootloader`.
+
+The API is public because applications that are booted by the immutable bootloader can call functions from this library via the bootloader's code, through external APIs.
+See :ref:`doc_fw_info_ext_api` for more information.
+
+The library provides the following functionality:
+
+* SHA256 hashing
+* SECP256R1 signature validation
+* Root-of-trust firmware validation, which is the function the bootloader uses to validate a firmware's signature and digest, using the SHA256 and SECP256R1 algorithms
+
+These functions are available as separate external APIs.
+The API can be used the same way regardless of which backend is used.
+
+Backends
+********
+
+When using the library, you can choose between the following backends:
+
+* Hardware backend :ref:`nrf_cc310_bl_readme` (can only be used if Arm CryptoCell CC310 is available)
+* Software backend :ref:`nrf_oberon_readme`
+* The backend that is configured by another image (which is used through an external API)
```

Comment:
It doesn't call the other image's backend directly.

How about this?

    * Another image's instance of bl_crypto, called via external APIs. The other image chooses its own backend.

Reply:
OK, updated.